lib/nuget-parser/index.ts

```diff
@@ -151,10 +160,24 @@ Supply a targetFramework by using the \x1b[1m--target-framework\x1b[0m argument.
   );
   const assemblyVersions = runtimeAssembly.generateRuntimeAssemblies(depsFile);
 
+  // Parse the TargetFramework using Nuget.Frameworks itself, instead of trying to reinvent the wheel, thus ensuring
```

Comment:
We need to do that when building the `depGraph`. The `project.assets.json` file can potentially have multiple lists of dependencies, one for each `TargetFramework`, and as I will display in a comment below, this is not stored uniformly in the assets file.